src/wallet/rpcwallet.cpp

```diff
@@ -1178,10 +1185,22 @@ UniValue ListReceived(const UniValue& params, bool fByAccounts)
     // Reply
     UniValue ret(UniValue::VARR);
     map<string, tallyitem> mapAccountTally;
-    BOOST_FOREACH(const PAIRTYPE(CBitcoinAddress, CAddressBookData)& item, pwalletMain->mapAddressBook)
+
+
```

Comment:
nit: prefer not to have two blank lines in the middle of a function. Perhaps you can move one of them to between the if block and for block below.